packages/app-builder/public/locales/en/scenarios.json

```diff
@@ -156,6 +156,14 @@
   "validation.evaluation_error.field_not_found_other": "{{count}} fields not found",
   "validation.evaluation_error.function_error_one": "function contains errors",
   "validation.evaluation_error.function_error_other": "{{count}} functions contain errors",
+  "validation.evaluation_error.null_field_read_one": "unexpected NULL field read",
+  "validation.evaluation_error.null_field_read_other": "{{count}} unexpected NULL field reads",
+  "validation.evaluation_error.no_rows_read_one": "data point is missing",
+  "validation.evaluation_error.no_rows_read_other": "{{count}} data points are missing",
+  "validation.evaluation_error.division_by_zero_one": "division by zero",
+  "validation.evaluation_error.division_by_zero_other": "{{count}} divisions by zero",
+  "validation.evaluation_error.payload_field_not_found_one": "payload field not found",
```

Comment:
I'm not sure when we have this vs just "null field read" 🤔

Reply:
I've looked into it, this is basically JSON specific (I guess):
- if key is `undefined` -> ErrPayloadFieldNotFound
- if key is `null`-> ErrNullFieldRead